usage-collection/src/generated/main/java/com/cloudera/thunderhead/service/common/usage/UsageProto.java

```diff
@@ -67503,6 +67503,10 @@ public CDPNetworkDetails parsePartialFrom(
     com.cloudera.thunderhead.service.common.usage.UsageProto.CDPEnvironmentTelemetryFeatureDetailsOrBuilder getTelemetryFeatureDetailsOrBuilder();
   }
   /**
+   * <pre>
```

Comment:
This file change does not really belong to the intended scope of the PR, but we can keep including it nevertheless.